packages/fx-core/src/plugins/solution/fx-solution/commonQuestions.ts

```diff
@@ -321,19 +321,32 @@ async function askCommonQuestions(
     }
     resourceGroupInfo = res.value;
   } else if (resourceGroupNameFromProfile && resourceGroupLocationFromProfile) {
-    const checkRes = await rmClient.resourceGroups.checkExistence(resourceGroupNameFromProfile);
-    if (checkRes.body) {
-      resourceGroupInfo = {
-        createNewResourceGroup: false,
-        name: resourceGroupNameFromProfile,
-        location: resourceGroupLocationFromProfile,
-      };
-    } else {
-      resourceGroupInfo = {
-        createNewResourceGroup: true,
-        name: resourceGroupNameFromProfile,
-        location: resourceGroupLocationFromProfile,
-      };
+    try {
+      const checkRes = await rmClient.resourceGroups.checkExistence(resourceGroupNameFromProfile);
```

Comment:
There is another checkExistence() call in this file. Is it deliberately left unchanged?

Reply:
I failed to find it. Could you please provide the code line? Thx